packages/ui/src/helpers/authenticator/formFields/defaults.ts

```diff
@@ -0,0 +1,168 @@
+/**
+ * This file contains helpers that generate default formFields for each screen
+ */
+import {
+  defaultFormFieldOptions,
+  getPrimaryAlias,
+  isAuthFieldWithDefaults,
+} from '..';
+import {
+  ActorContextWithForms,
+  AuthMachineState,
+  FormFields,
+  FormField,
+  FormFieldComponents,
+  SignInState,
+  SignInContext,
+} from '../../../types';
+import { getActorContext, getActorState } from '../actor';
+
+/** Helper function that gets the default formField for given field name */
+const getDefaultFormField = (
+  state: AuthMachineState,
+  fieldName: keyof typeof defaultFormFieldOptions
+) => {
+  const { country_code } = getActorContext(state) as ActorContextWithForms;
+  let options: FormField = defaultFormFieldOptions[fieldName];
+  const { type } = options;
+
+  if (type === 'tel') {
+    options = { ...options, dialCode: country_code };
+  }
+
+  return options;
+};
+
+/** Helper function that returns default form field for configured primary alias */
+const getAliasDefaultFormField = (state: AuthMachineState): FormField => {
+  const primaryAlias = getPrimaryAlias(state);
+  return {
+    ...getDefaultFormField(state, primaryAlias),
+    autocomplete: 'username',
+  };
+};
+
+/** Reusable confirmation code form fields. */
+const getConfirmationCodeFormFields = (
+  state: AuthMachineState
+): FormFields => ({
+  confirmation_code: {
+    ...getDefaultFormField(state, 'confirmation_code'),
+    label: 'Code *',
+    placeholder: 'Code',
+  },
+});
+
+const getSignInFormFields = (state: AuthMachineState): FormFields => ({
+  username: { ...getAliasDefaultFormField(state) },
+  password: {
+    ...getDefaultFormField(state, 'password'),
+    autocomplete: 'current-password',
+  },
+});
```

Comment:
This translates to:

`signIn` has default formFields with
1. primaryAlias as username, and
2. password with autocomplete set to `current-password`